draft-packages/tooltip/KaizenDraft/Tooltip/Tooltip.tsx

```diff
@@ -145,6 +156,39 @@ const TooltipContent = ({
   ) : null
 }
 
+const isSemanticElement = (
+  elementType: string | JSXElementConstructor<any>
```

Comment:
It's not `JSX.Element`?

Reply:
At this point its been narrowed to the `type` property of a `React.ReactPortal | React.ReactElement<unknown, string | React.JSXElementConstructor<any>>` which, assuming TS isn't telling porky pies, should be either and `string` or `JSXElementConstructor<any>`

![Screenshot 2023-07-05 at 12 32 03 pm](https://github.com/cultureamp/kaizen-design-system/assets/36558508/f5d52d62-933d-4058-847d-9ff4315df5eb)